### ISSUE TITLE
Newly purchased EWRs are showing as degraded (yellow bar)

### ISSUE THREAD

**Issue by Starfire13** [dcs-liberation/dcs_liberation#2142]
### Affected versions

Development build

### Development build

https://github.com/dcs-liberation/dcs_liberation/actions/runs/2066591311

### Description

Newly purchased EWRs are showing a yellow bar beneath the icon.

![Capture](https://user-images.githubusercontent.com/72491792/161051577-1cb1b90a-c830-42fc-b932-781d2220a328.JPG)

### Save game and other files

[Vegas Nerve #3751.zip](https://github.com/dcs-liberation/dcs_liberation/files/8389178/Vegas.Nerve.3751.zip)

**Comment by Starfire13** [comment]
Note that this is an EWR purchased at a SAM site. Should we be able to convert SAM sites into EWRs?

**Comment by RndName** [comment]
> Note that this is an EWR purchased at a SAM site. Should we be able to convert SAM sites into EWRs?

Yeah this causing the yellow bar because the game thinks this is a threat but has nothing to threat you with as it has no launchers. Therefore its degraded. 

Your question is the right question! IMO this should be possible, why mnot?

I may change the degraded calculation logic to actually check if the thing is threat or not. Aka check if there was a launcher. If there was none this should not show as degaded. If there was one and it is destroyed -> degraded

**Comment by DanAlbert** [comment]
If we're replacing a SAM TGO group with an EWR without changing its type other things will be broken. The AI commander will treat it as a SAM because that's its type.
